src/client/minimap.h

```diff
@@ -78,6 +78,7 @@ struct MinimapData {
 	video::IImage *minimap_mask_round = nullptr;
 	video::IImage *minimap_mask_square = nullptr;
 	video::ITexture *texture = nullptr;
+	bool textures_initialised = false; // True if the following textures are not nullptrs.
 	video::ITexture *heightmap_texture = nullptr;
```

Comment:
Oh, yes, you are right, ~~I moved it one line below~~, since `heightmap_texture` does also get initialized by `getMinimapTexture()` even though it only returns `minimap_texture`. 
https://github.com/minetest/minetest/blob/e269a7963e27092e3f12b0529e8feb5ee78fe79c/src/client/minimap.cpp#L583
https://github.com/minetest/minetest/blob/e269a7963e27092e3f12b0529e8feb5ee78fe79c/src/client/minimap.cpp#L613-L614

You already fix it, so I guess this is resolved.